scripts/list-outdated-dependencies/mobile-dependencies.txt

```diff
@@ -1,5 +1,6 @@
 @babel/plugin-transform-export-namespace-from
 @config-plugins/detox
+@expo/config-plugins
```

Comment:
only `@types/react`, `react` and `react-dom` added. Everything else is just sorting